test/org/loklak/harvester/RedirectUnshortenerTest.java

```diff
@@ -21,7 +21,7 @@ public void testRedirectUnshortener() {
         shortlinkMap.put("http://ow.ly/P9SX30ddSTI", "https://github.com/loklak/loklak_server/issues/1284");
         shortlinkMap.put("http://bit.do/blog-fossasia", "http://blog.fossasia.org/");
         shortlinkMap.put("http://fb.me/4lcXZsyyO", "https://www.facebook.com/permalink.php?story_fbid=1550262581900846&id=1381813572079082");
-        shortlinkMap.put("http://wp.me/sf2B5-shorten", "https://en.blog.wordpress.com/2009/08/14/shorten/");
+        shortlinkMap.put("http://wp.me/sf2B5-shorten", "http://en.blog.wordpress.com/2009/08/14/shorten/");
```

Comment:
No I don't seem to understand the statement, based on a URL shortened how can you say that it is a threat to anonymity or that it is a trackable or untrackable URL? if i have an URL which is http://b.ly vs https://b.ly where is the anonymity coming into place?

At what stage of URL redirect solution do you decide that the URL is not `anonymized`. Either way when your request hits the URL it will be tracked by who so ever does it. I don't see anonymity threats anywhere here.

My question is why does this test have `http://` instead of `https://` when `https://` works? Does the test fail when it's https?